transactions/pendingtxtracker.go

```diff
@@ -24,12 +25,14 @@ import (
 )
 
 const (
-	// EventPendingTransactionUpdate is emitted when a pending transaction is updated (added or deleted). Carries StatusChangedPayload in message
+	// EventPendingTransactionUpdate is emitted when a pending transaction is updated (added or deleted). Carries PendingTxUpdatePayload in message
```

Comment:
curious, does this change break the current api? i.e you will have to make updates to desktop for this to work with the current feature?

Reply:
It doesn't. For now, we still use the deprecated `watch` API https://github.com/status-im/status-go/blob/405d468e0e74e78a89f0653ad6514c50af9d1d6f/services/wallet/api.go#L283